config/datocms/migrations/1730405111_linkRecords.ts

```diff
@@ -0,0 +1,150 @@
+import { Client } from '@datocms/cli/lib/cma-client-node';
+
+export default async function (client: Client) {
+  console.log('Update existing fields/fieldsets');
+
+  console.log(
+    'Update Structured text field "Text" (`text`) in block model "\uD83D\uDCDD \uD83D\uDDBC\uFE0F Text Image Block" (`text_image_block`)'
+  );
+  await client.fields.update('V4dMfrWsQ027JYEp6q3KhA', {
+    validators: {
+      required: {},
+      structured_text_blocks: {
+        item_types: [
+          'ZdBokLsWRgKKjHrKeJzdpw',
+          'gezG9nO7SfaiWcWnp-HNqw',
+          '0SxYNS2CR1it_5LHYWuEQg',
+        ],
+      },
+      structured_text_links: {
+        on_publish_with_unpublished_references_strategy: 'fail',
+        on_reference_unpublish_strategy: 'delete_references',
+        on_reference_delete_strategy: 'delete_references',
+        item_types: ['LjXdkuCdQxCFT4hv8_ayew', 'X_tZn3TxQY28ltSyjZUGHQ'],
+      },
+    },
+  });
+
+  console.log(
```

Comment:
Also used this occasion to move the `slug` field back to the top. Is was dangling all the way at the bottom of the page.